editor/plugins/control_editor_plugin.cpp

```diff
@@ -648,11 +679,11 @@ AnchorPresetPicker::AnchorPresetPicker() {
 	extra_row->add_theme_constant_override("separation", grid_separation);
 	main_vb->add_child(extra_row);
 
-	_add_row_button(extra_row, PRESET_LEFT_WIDE, TTR("Left Wide"));
-	_add_row_button(extra_row, PRESET_VCENTER_WIDE, TTR("VCenter Wide"));
-	_add_row_button(extra_row, PRESET_RIGHT_WIDE, TTR("Right Wide"));
+	_add_row_button(extra_row, PRESET_LEFT_WIDE, TTRC("Left Wide"));
+	_add_row_button(extra_row, PRESET_VCENTER_WIDE, TTRC("VCenter Wide"));
+	_add_row_button(extra_row, PRESET_RIGHT_WIDE, TTRC("Right Wide"));
 	_add_separator(extra_row, memnew(VSeparator));
-	_add_row_button(extra_row, PRESET_FULL_RECT, TTR("Full Rect"));
+	_add_row_button(extra_row, PRESET_FULL_RECT, TTRC("Full Rect"));
```

Comment:
Would be neat if the code for anchor presets now present in `Control._bind_methods()` was somewhat consolidated with these buttons in the editor plugin, so that the translation strings are less duplicated around.

Not something that should be done in this PR, just food for thought.